src/webview.css

```diff
@@ -41,30 +39,33 @@ circle {
 
 .current-note {
   fill: var(--distance-0-primary-color);
-  stroke: var(--distance-0-secondary-color);
   r: 18;
 }
 
 line {
   stroke: var(--distance-remaining-primary-color);
 }
 
-line.distance-0 {
+.adjacent-line {
   stroke: var(--distance-0-primary-color);
   stroke-width: 3px;
 }
 
-line.distance-1 {
-  stroke: var(--distance-1-primary-color);
+.node-label {
+  fill: var(--joplin-color);
 }
 
-.adjacent-line {
-  stroke: var(--distance-0-primary-color);
-  stroke-width: 3px;
+.hovered-node {
+  fill: var(--joplin-color-error) !important;
```

Comment:
I love the changes, but my only comment is that the red error color seems aggressive for the graph.  At least in the Light and Dark themes I tried it in.

What are your thoughts on using something like `joplin-color-correct` or `joplin-color-warn`?    i.e. green or yellow

Reply:
after testing locally, I think I prefer green (`joplin-color-correct`).

Reply:
Yes, that is a great idea.
Currently, there are two colors used: one for the nodes with more contrast and one for rest (label, links, arrows).

I found something like #93f500 to be good for fomer, and `joplin-color-correct` with latter. So let's change it?

Reply:
Ideally I would like to let users them selves change styles by creating an additional menu entry for custom CSS - that could be PR in future

Reply:
Let’s go with those new colors you suggested.

Yeah an interface for customizing colors in future would be awesome!

Reply:
Nice, thanks for the quick answer! Allright, changes done.
When having time, I'll probably give a short intro of new features in the discourse forum (or you, if you want)

Reply:
Looks good, merged!   I just kicked off a release, definitely feel free to shout out the new features. I did something similar a couple days ago but you can do the same